### DISCUSSION TITLE
About | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#63] [General]
https://oliverlew.github.io/PAT/about.html 

 浙江大学编程能力测试(PAT)C语言题解说明

**Comment by javioustlj** [comment]
请问你这个blog是怎么实现的呢？我的问题详细描述如下
我看你的oliverlew.github.io这个仓库中并没有任何的文章，请问你的文章是放在哪里呢？Github Pages的博客，文章不是需要放在oliverlew.github.io这个仓库里面吗？请问你是怎么做的？

**Comment by xlucn** [comment]
> @javioustlj
> 请问你这个blog是怎么实现的呢？我的问题详细描述如下
> 我看你的oliverlew.github.io这个仓库中并没有任何的文章，请问你的文章是放在哪里呢？Github Pages的博客，文章不是需要放在oliverlew.github.io这个仓库里面吗？请问你是怎么做的？

我的博客内容在单独的[repo](https://github.com/OliverLew/PAT/tree/gh-pages)里，github.io目前只有issues

**Comment by wuxueqian14** [comment]
```c
cout << "Your blog is pretty good!";
```
Your GitHub Pages gives me a new look.Beautiful!

**Comment by TricoChan** [comment]
艳羡博主清晰简洁的思维逻辑，向你学习！
